syne_tune/optimizer/baselines.py

```diff
@@ -27,6 +27,16 @@ def __init__(self, config_space: Dict, metric: str, **kwargs):
         )
 
 
+class GridSearch(FIFOScheduler):
+    def __init__(self, config_space: Dict, metric: str, **kwargs):
```

Comment:
Can you add comment on what the method does? (they are things to say in the design, how configurations are ordered and in particular that only categorical is supported for now)